recipes/main/thai_red_curry.md

```diff
@@ -0,0 +1,41 @@
+# Thai red curry
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-30%25-green.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-1.5hr-orange.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Average-yellow.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+**Head chef:** [@matthewdking](https://github.com/matthewdking/)
+
+**Sous chef:** [@Karyum](https://github.com/Karyum/)
+
+**Date:** 29/01/18, Week 13 FACN3
+
+**Number of servings:** 8
+
+**Author of Recipe**: [@matthewdking](https://github.com/matthewdking/)
+
+## Ingredients
+
+* 2 onions, sliced
+* 4 small sweet potatoes, peeled and chopped into about 2cm cubes
+* 1 large aubergine, chopped into about 2cm cubes
+* 2 red peppers, sliced
+* thai red curry paste
```

Comment:
how much here is important to determine how spicy.
what if i never used a curry paste  ( i literally never cooked with a curry paste ) how would i know how much i should use ?

Reply:
I have defined the specific amounts for this in the recipe because it varies depending on taste? Maybe I should change it up here as well though